things/mocks/commons.go

```diff
@@ -83,3 +86,15 @@ func sortChannels(pm things.PageMetadata, chs []things.Channel) []things.Channel
 
 	return chs
 }
+
+func parseID(ID string) (id uint64) {
+	var slNum string
```

Comment:
What does `slNum` stands for?

Reply:
I guess it is serialNum? I have changed it to 'serialNum'.